assemblies/dynamic-plugins/assembly-install-third-party-plugins-rhdh.adoc

```diff
@@ -0,0 +1,32 @@
+[id="assembly-install-third-party-plugins-rhdh"]
+= Installing third-party plugins in {product}
+:context: assembly-install-third-party-plugins-rhdh
+
+You can install a third-party plugins in {product} without rebuilding the {product-very-short} application.
+
+The location of the `dynamic-plugin-config.yaml` file depends on the deployment method. For more details, refer to xref:proc-config-dynamic-plugins-rhdh-operator_rhdh-installing-rhdh-plugins[] and xref:con-install-dynamic-plugin-helm_rhdh-installing-rhdh-plugins[].
```

Comment:
```suggestion
The location of the `dynamic-plugin-config.yaml` file depends on the deployment method. For more details, refer to xref:proc-config-dynamic-plugins-rhdh-operator_rhdh-installing-rhdh-plugins[Installing dynamic plugins with the {product} Operator] and xref:con-install-dynamic-plugin-helm_rhdh-installing-rhdh-plugins[Installing dynamic plugins using the Helm chart].
```
These links are broken in the GitHub preview...I'm not sure if adding the section heading will resolve the issue.